### DISCUSSION TITLE
EP/0336 CANNOT ADD/DELETE PRISMA ESTIMATE INSERTIONS

### DISCUSSION THREAD

**Discussion by Tiago139** [prisma/prisma#12722] [Q&A] [answered]
I have been trying to add insertions into Print Buy Tool Kit but I have been greeted by an error message that read:
"EP/0336 CANNOT ADD/DELETE PRISMA ESTIMATE INSERTIONS"
![DDS Error Message](https://user-images.githubusercontent.com/103199905/162223026-7fade283-1884-4822-9700-837e64ef9c5e.png)

**Comment by janpio** [comment, marked as answer]
Hey @Tiago139, I don't think we are the correct Prisma for this. This Prisma here is a ORM for databases in the Node.js ecosystem. Sorry we can not help you with this request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
